www/docs/reactjs/useQueries.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: usequeries
+title: useQueries()
+sidebar_label: useQueries()
+slug: /use-queries
+---
+
+The useQueries hook is used to fetch multiple queries using one hook call. This hook is an excellent choice in scenarios where you fetch numerous queries in one component as it can lead to a cleaner and more readable code.
```

Comment:
Is this really what `useQueries` is for? cc @TkDodo, I'm unsure even after reading the docs page

Reply:
The main use-case would be for a variable amount of queries, usually of the same type. So, fetch a list of todos, then fetch all details for all todos in a `useQueries` where you `map` over the result of the first query.

having one `useQueries` where you fetch two different resources with different types is certainly possible, but there isn't much advantage over calling `useQuery` twice I believe, unless you use `suspense`, because `useQueries` can trigger suspense in parallel, which `useQuery` cannot (it would waterfall).

Reply:
@ixahmedxi could we get the examples to reflect this?

i.e.

```ts
trpc.useQueries(t => postIds.map(id => t.post.byId({ id })))
```